Fitbit.Portable/JsonDotNetSerializerExtensions.cs

```diff
@@ -9,6 +9,23 @@ namespace Fitbit.Api.Portable
     internal static class JsonDotNetSerializerExtensions
     {
         /// <summary>
+        /// Parses the error structure which is common when errors are raised from the api
+        /// </summary>
+        /// <param name="serializer"></param>
+        /// <param name="errorJson"></param>
+        /// <returns></returns>
+        internal static List<ApiError> Errors(this JsonDotNetSerializer serializer, string errorJson)
```

Comment:
@WestDiscGolf I think this should be named `ParseFitbitErrors` or something similar. Can you amend the PR?